backend/databases/sql.py

```
@@ -0,0 +1,216 @@
+
+def create_user(username: str, password: str, email: str, phone_number: str, full_name: str, address: str):
+    """Create new user - CONTAINS DELIBERATE SQL INJECTION"""
+    conn = sqlite3.connect(DATABASE_PATH)
+    cursor = conn.cursor()
+    
+    try:
+        # DELIBERATE SQL INJECTION - User input directly in query
+        insert_query = f"""
+            INSERT INTO users (username, password, email, phone_number, full_name, address)
+            VALUES ('{username}', '{password}', '{email}', '{phone_number}', '{full_name}', '{address}')
+        """
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)